website/docs/en/config/output/manifest.mdx

```diff
@@ -0,0 +1,47 @@
+# output.manifest
+
+- **Type:** `string | boolean`
+- **Default:** `false`
+
+Whether to generate a manifest file that contains information of all assets, and the mapping relationship between [entry module](/config/source/entry) and assets.
+
+When set to `true`, Rsbuild will generate a `manifest.json` file. When the value is a `string`, it will be used as the manifest file name.
```

Comment:
```suggestion
When set to `true`, Rsbuild will generate a `manifest.json` file after building. When the value is a string, it will be used as the manifest file name.
```